resources/resources.qrc

```diff
@@ -92,5 +92,6 @@
         <file>icons/Star.svg</file>
         <file>icons/Rectangle7.png</file>
         <file>icons/Rectangle8.png</file>
+        <file>icons/xby.png</file>

```

Comment:
This reference to `xby.png` was added but the file itself doesn't seem to be here, causing a "missing file" error during compilation.

Reply:
Sorry about that, file added - please retry!